mongodb_base_block.py

```diff
@@ -84,6 +89,9 @@ class MongoDBBase(Block):
     database = StringProperty(title='Database Name', default="test")
     collection = ExpressionProperty(title='Collection Name', default="signals")
     creds = ObjectProperty(Credentials, title='Credentials')
+    mongo_version = SelectProperty(MongoVersion,
+                                   default=MongoVersion.v2,
+                                   title="MongoDB Version")
```

Comment:
Doesn't this v2 vs. v3 issue have to do with the pymongo version, not the MongoDB version? If so, is their a way to inspect the version of pymongo and not have to do this as a user config property?